enterprise/backend/src/metabase_enterprise/data_editing/data_editing.clj

```diff
@@ -47,14 +47,27 @@
   "Inserts rows into the table, recording their creation as an event. Returns the inserted records.
   Expects rows that are acceptable directly by [[actions/perform-action!]]. If casts or reversing coercion strategy
   are required, that work must be done before calling this function."
-  [table-id rows]
-  (let [res (perform-bulk-action! :bulk/create table-id rows)]
-    (when-some [user-id api/*current-user-id*]
```

Comment:
I've heard it said that "you when-some, you lose some", and I don't want to lose any snapshots.